### DISCUSSION TITLE
Is it possible to update the filter in a content filter topic

### DISCUSSION THREAD

**Discussion by tmayoff** [OpenDDS/OpenDDS#3813] [Q&A] [answered]
Can you update a Content Filter Topic's filter (or just update the filter's query parameters) without having to recreate it

**Comment by tmayoff** [comment]
Nevermind, I immediately found it after posting this, I was looking in the wrong place. I don't think I can remove the discussion topic

**Comment by mitza-oci** [comment, marked as answer]
Filter expressions are immutable.  As you noted, query parameters can change at any time.
